### PR TITLE
[GST-2195] Travis change to include Ansible 2.1.1.0

### PR DESCRIPTION
**Pull request by quater**
Let's see what breaks so I can implement fixes.

### CODE REVIEW

**Comment by quater** [comment]
The Travis test was successful for Ansible 2.1.1.0 and thus no additional code changes are required.

File: tests/vagrant/integration_requirements.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 
 - src: sansible.java
-  version: v1.0
+  version: v1.0.3
```
Comment (lobsterdore) [inline tests/vagrant/integration_requirements.yml]: Just need to remove the patch versions here.

Comment (quater) [reply tests/vagrant/integration_requirements.yml]: Changed accordingly

**Comment by lobsterdore** [comment]
LGTM :+1:

**Pull request merged**